### PR TITLE
Add type hints to random

### PR DESCRIPTION
**Pull request by souravsingh**
Fixes a part of #808

### CODE REVIEW

File: axelrod/random_.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 from axelrod import Actions
 
 
-def random_choice(p=0.5):
+def random_choice(p=0.5: float):
```
Comment (Nikoleta-v3) [inline axelrod/random_.py]: Hello @souravsingh I am a member of the Axelrod team :smile:!
I believe you should drop `:float` cause `p` is determined. That should be the syntax error

Comment (marcharper) [inline axelrod/random_.py]: I think the syntax parser wants something like `(p: float = 0.5)`

Comment (souravsingh) [inline axelrod/random_.py]: @Nikoleta-v3 Nice to meet you! 
I will look into the review.

@marcharper What should be the return type set? Should it be imported as Action.

Comment (marcharper) [inline axelrod/random_.py]: Yeah importing the type from `actions.py` is the way to go.

**Comment by souravsingh** [comment]
The PR is now ready for merging.

**Pull request merged**